package.json

```diff
@@ -97,6 +104,9 @@
     "@opentelemetry/instrumentation-net": "~0.16.0",
     "@opentelemetry/instrumentation-pg": "~0.16.0",
     "@opentelemetry/instrumentation-redis": "~0.16.0",
+    "@opentelemetry/instrumentation-bunyan": "~0.16.0",
+    "@opentelemetry/instrumentation-pino": "~0.16.0",
+    "@opentelemetry/instrumentation-winston": "~0.16.0",
```

Comment:
I think these should be added to `peerDependenciesMeta` section below as well.

Reply:
Thanks, fixed